src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -48,7 +46,7 @@
 ERR_INSUFFICIENT_ROLE = "You cannot remove a member who has more access than you."
 ERR_INSUFFICIENT_SCOPE = "You are missing the member:admin scope."
 ERR_MEMBER_INVITE = "You cannot modify invitations sent by someone else."
-ERR_MEMBER_REINVITE = "You can only reinvite members; you cannot modify other member details."
+ERR_EDIT_WHEN_REINVITING = "You cannot modify other member details when resending an invitation."
```

Comment:
This error doesn't show to our frontend ever and only API users will see it. What are they supposed to do if they want to modify the invite and resend? Should we add that context to the error message too?

Reply:
Would it be better to add something like "Separate requests must be made to modify member details and reinvite members"?